buildSrc/src/main/java/org/elasticsearch/gradle/vagrant/VagrantMachine.java

```diff
@@ -198,7 +210,7 @@ protected void logLine(String line) {
             if (progress != null) {
                 progressLogger.progress(progress);
             }
-            System.out.println(line);
```

Comment:
This was actually leftover from debugging. I don't think we want this always going to stdout. In fact, this ProgressOutputStream is meant only to write to the progress logger.